client/src/Molecules.elm

```diff
@@ -5,29 +5,36 @@ module Molecules exposing
     , ChangeMol(..)
     , Level0
     , Level1
+    , Vector(..)
     , allOverhangs
     , backboneDecoder
+    , backboneEncoder
     , initBackbone
     , initLevel0
     , initLevel1
     , interpretBackboneChange
     , interpretLevel0Change
     , level0Decoder
+    , level0Encoder
     , level1Decoder
+    , levelNEncoder
```

Comment:
Can you write a `vectorEncoder` that dispatches to these as necessary please?

Reply:
Taken care of in the latest commit.